.dockerignore

```diff
@@ -0,0 +1,14 @@
+.venv
+.flake8
+.gitignore
+checklist.md
+.githab

```

Comment:
There seems to be a typo in the file name `.githab`. If this is intended to ignore GitHub-related directories, it should be `.github`.